.github/workflows/run_petprep.yml

```
@@ -0,0 +1,74 @@
+name: Run PETPREP
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
+    tags: "*"
+  pull_request:
+
+env:
+  # for local/self hosted runners one can assume that the 
+  FREESURFER_HOME: /usr/local/freesurfer/7.4.1
+  FSLDIR: /opt/conda/envs/petprep
+  FSL_OUTPUT_TYPE: NIFTI_GZ
+  AFNI_PATH: /opt/afni-latest
+  FSL_PATH: /opt/conda/envs/petprep/share/fsl/bin
+  ANTS_PATH: /opt/ants-2.6.2/bin
+  WORKBENCH_PATH: /opt/workbench/exe_linux64
+
+jobs:
+  collect-data:
+    uses: ./.github/workflows/collect_data.yml
+
+  run-tests:
+    needs: collect-data
+    runs-on: local
+    env:
+      DATA_DIR: ${{ needs.collect-data.outputs.data_directory }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 200
+          fetch-tags: true
+      
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      
+      - name: Install uv
+        run: |
+          curl -LsSf https://astral.sh/uv/install.sh | sh
+          echo "$HOME/.cargo/bin" >> $GITHUB_PATH
+
+      - name: Build and Install PETPREP
+        run: |
+          uv build 
+          pip install dist/*.whl --force-reinstall
+
+      - name: Check petprep is installed
+        run: |
+          which petprep
+          petprep --version
+
+      - name: Verify test data exists
+        run: |
+          if [ ! -d "$DATA_DIR" ] || [ -z "$(ls -A $DATA_DIR 2>/dev/null)" ]; then
+            echo "Error: Test data directory is empty or missing"
+            exit 1
+          fi
+          echo "Test data verified at $DATA_DIR"
+          ls -la "$DATA_DIR"
+
+      - name: Run petprep
+        run: |
+          source "${FREESURFER_HOME}/SetUpFreeSurfer.sh"
+          export PATH="${AFNI_PATH}:${PATH}"
+          export PATH="${FSL_PATH}:${PATH}"
+          export PATH="${ANTS_PATH}:${PATH}"
+          export PATH="${WORKBENCH_PATH}:${PATH}"
+          export FSLDIR="${FSLDIR}"
+          export FSL_OUTPUT_TYPE="${FSL_OUTPUT_TYPE}"
+          petprep "${DATA_DIR}" "${DATA_DIR}/derivatives/petprep" participant --fs-license-file "${FREESURFER_HOME}/license.txt" --sloppy --derivatives anat=${DATA_DIR}/derivatives/smriprep --fs-subjects-dir ${DATA_DIR}/derivatives/freesurfer 
```

Maybe want to use `--fs-no-resume` to avoid rerunning?

```suggestion
          petprep "${DATA_DIR}" "${DATA_DIR}/derivatives/petprep" participant \
            --fs-license-file "${FREESURFER_HOME}/license.txt" \
            --fs-subjects-dir ${DATA_DIR}/derivatives/freesurfer --fs-no-resume \
            --sloppy --derivatives anat=${DATA_DIR}/derivatives/smriprep
```

---

We should do what fMRIPrep is doing :-) @effigies 

If the processing has access to motion corrected data and FreeSurfer derivatives it shouldn't take more than an hour to run.